### DISCUSSION TITLE
Installing rastervision via pip fails

### DISCUSSION THREAD

**Discussion by vikamayr** [azavea/raster-vision#1841] [Q&A]
I have tried to install rastervision with pip on WSL2, but I seem to get error which relates to pygeos=0.13 version.

```
Collecting pygeos==0.13 (from rastervision-core==0.20.2->rastervision)
....error: subprocess-exited-with-error.
```
Problem seems to relate with that pygeos==0.13 version as later version pygeos==0.14 installation works.

**Comment by AdeelH** [comment]
Thanks for reporting. As a workaround, you can try installing `rastervision_core` [without its dependencies](https://stackoverflow.com/a/12759996/5908685) and then installing the [dependencies](https://github.com/azavea/raster-vision/blob/0.20/rastervision_core/requirements.txt) separately, changing the `pygeos` version.

**Comment by AdeelH** [comment]
FYI, the newly released Raster Vision v0.21 does not have the `pygeos` dependency. Consider giving that a try.
